nixos/modules/services/x11/desktop-managers/gnome3.nix

```diff
@@ -160,15 +160,9 @@ in
         customSessions = mkOption {
           type = types.listOf (types.submodule {
             options = {
-              wmName = mkOption {
```

Comment:
you make to use mkRemovedOption~

Reply:
Unfortunately I don't think that works because the option is in a submodule. I added an assertion.